### PR TITLE
Removed usa-search.js from header.html

### PR DESCRIPTION
**Pull request by aubreyarcangel**
## Description
- Removed code in `usa-search.js` script for search.usa.gov 
Renamed `usa-search.js` to `incompatible-browser.js` because the remaining code in the file checks if the user's browser is IE10 or older, if so, it displays the `incompatible-browser-warning` in top-nav.html

## Testing done
Tested search on local instance and Saucelabs.com windows IE10 browser.

## Screenshots
Made sure incompatible browser warning still displays after renaming the file. 

screen shot 2019-01-15 at 12 44 57 pm

## Acceptance criteria
- [X] No errors when trying to run search
- [X] Incompatible browser warning displays with IE10 or older

## Definition of done
- [X] Events are logged appropriately
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [X] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ncksllvn — APPROVED** [review]
LGTM, but deferring to @bshyong to confirm that we definitely don’t need this anymore. If not, can we also delete the file from the assets directory?

File: src/site/includes/header.html [hunk 1]
```diff
@@ -116,5 +116,3 @@ <h3 class="usa-alert-heading">{{ banner.title }}</h3>
     </div>
   </div>
 {% endif %}
-
-<script async src="/js/usa-search.js"></script>
```
Comment (ncksllvn) [inline src/site/includes/header.html]: As part of cleaning up, can you remove this file from `src/site/assets/js`?

Comment (aubreyarcangel) [reply src/site/includes/header.html]: deleted!

File: src/site/assets/js/usa-search.js [hunk 2]
```diff
@@ -1,22 +0,0 @@
-var usasearch_config = { siteHandle:"vets.gov_search" };
-var script = document.createElement("script");
-script.src = "https://search.usa.gov/javascripts/remote.loader.js";
-script.async = true;
-document.getElementsByTagName("head")[0].appendChild(script);
-
-(function() {
```
Comment (bshyong) [inline src/site/assets/js/usa-search.js]: I believe this function should not be removed. It is used to detect incompatible browsers and display a warning message located in `top-nav.html`. You could remove lines 1-5 and rename this script file to more accurately reflect the contents

**Review by bshyong — DISMISSED** [review]

**Comment by ncksllvn** [comment]
Thank you @bshyong!

File: src/site/assets/js/usa-search.js (hunk 2 above)
Comment (ncksllvn) [reply src/site/assets/js/usa-search.js]: Makes perfect sense. @aubreyarcangel, when you have bandwidth, can you make that update per @bshyong's comment? A name like `incompatible-browser.js` or something like that might really help describe what that script does.

Comment (ncksllvn) [reply src/site/assets/js/usa-search.js]: That is perfect - thank you!

File: src/site/includes/header.html [hunk 3]
```diff
@@ -117,4 +117,4 @@ <h3 class="usa-alert-heading">{{ banner.title }}</h3>
   </div>
 {% endif %}
 
-<script async src="/js/usa-search.js"></script>
+<script async src"/js/incompatible-browser.js"></script>
```
[the remark this answers is not in the document]
Comment (aubreyarcangel) [reply src/site/includes/header.html]: Fixed! Good catch, thank you!

**Review by ncksllvn — APPROVED** [review]
LGTM! Great updates. One less HTTP request and script execution 🔥

**Pull request merged**